Sources/MapboxDirections/VisualInstructionComponent.swift

```diff
@@ -254,6 +254,7 @@ public struct GuidanceViewImageRepresentation: Equatable {
 extension VisualInstruction.Component: Codable {
     private enum CodingKeys: String, CodingKey {
         case kind = "type"
+        case subType = "subType"
```

Comment:
Is the string literal needed? Since the case name is the same as the literal.

Reply:
Still unclear if we should name it as `subType`, maybe `subKind` will align better? I'll remove custom coding key when we decide how to name it.

Reply:
How I see it:
- `subType` works well because it is the same as what is seen in the response, and this naming is clear.
- `subKind` aligns with the other coding key, so it's good for consistency. However, this naming is a bit clunky (imo), and it might be confusing that this `subKind` doesn't work for all `kind`s.

Ideally, we would use `type` and `subType` here imo, but since that would be a breaking change, I would say `subKind` is better for consistency's sake. The documentation for `SubType` below is clear that this is solely for guidance views.

Reply:
Will leave `subType` as undocumented for now.